src/elements/content-sidebar/SidebarNav.js

```diff
@@ -65,6 +70,16 @@ const SidebarNav = ({
         <div className="bcs-SidebarNav" aria-label={intl.formatMessage(messages.sidebarNavLabel)}>
             <div className="bcs-SidebarNav-tabs">
                 <SidebarNavTablist elementId={elementId} isOpen={isOpen} onNavigate={onNavigate}>
+                    {hasBoxAI && (
+                        <SidebarNavButton
+                            data-resin-target={SIDEBAR_NAV_TARGETS.BOXAI}
+                            data-testid="sidebarboxai"
+                            sidebarView={SIDEBAR_VIEW_BOXAI}
+                            tooltip={intl.formatMessage(messages.sidebarBoxAITitle)}
+                        >
+                            <BoxAiLogo height={Size5} width={Size5} />
```

Comment:
❓ have you tested that these sizes are rendered correctly in Safari? I've had unexpected behavior with these SVGs and height/width props there